### ISSUE TITLE
Sequence prediction for many to one model

### ISSUE THREAD

**Issue by abhijithneilabraham** [keras-team/keras#11256]
I am trying to find the output value for a sequence of numbers ranging from 1-5(5 timesteps). I am not sure of which loss function to use here during model.compile?

**Comment by harshini-gadige** [comment]
Hi

This issue isn't related to a bug/enhancement/feature-request or other accepted types of issue. 

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!
